config/application.example.yml

```diff
@@ -44,6 +44,10 @@
 # EducationProgram extension (DEPRECATED).
   enable_legacy_courses: 'false'
 
+# Set the default course type for newly created courses.  Use either
+# "ClassroomProgramCourse" or "BasicCourse".
```

Comment:
Thanks, I wasn't clear enough.  I was trying to suggest that there were two reasonable defaults, but you're right, no reason to be so inflexible!